services/kik.yml

```
@@ -3,3 +3,4 @@ name: Kik
+group: hosting
```

https://en.wikipedia.org/wiki/Kik_Messenger
```suggestion
group: messenger
```